modules/seqexec/server/src/main/scala/seqexec/server/gems/GemsControllerSim.scala

```diff
@@ -2,36 +2,33 @@
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
 package seqexec.server.gems
-import cats.effect.Sync
+
+import cats.Applicative
 import cats.implicits._
+import io.chrisdavenport.log4cats.Logger
 import seqexec.server.tcs.Gaos.{PauseConditionSet, PauseResume, ResumeConditionSet}
-import org.log4s.getLogger
 import seqexec.server.gems.Gems.{GemsWfsState, Cwfs1DetectorState, Cwfs2DetectorState, Cwfs3DetectorState, Odgw1DetectorState, Odgw2DetectorState, Odgw3DetectorState, Odgw4DetectorState}
 import seqexec.server.gems.GemsController.GemsConfig
 
-final class GemsControllerSim[F[_]: Sync] private extends GemsController[F] {
```

Comment:
But, but, I do like the explicit class. I only like anonymous classes for small definitions.

Reply:
Ok, I can revert that

Reply:
No need, it's just a matter of preferences.